parm/config/config.efcs

```diff
@@ -18,6 +18,7 @@ if [ $QUILTING = ".true." ]; then
     export npe_efcs=$npe_fv3
 fi
 
+export ntasks_efcs=$npe_efcs
```

Comment:
Why are we introducing a new variable that is exactly the same?

Reply:
I update it similar with config.fcst. In order to pass ntasks to env file.

Reply:
addressed